Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -809,17 +809,20 @@ def sort_all_list_incident_fields(incident_data):
 
     if incident_data.get('alerts', []):
         incident_data['alerts'] = sort_by_key(incident_data.get('alerts', []), main_key='alert_id', fallback_key='name')
-        reformat_sublist_fields(incident_data['alerts'])
+        if not argToBoolean(demisto.params().get('dontFormatSublists', False)):
```

Comment:
Instead of doing `argToBoolean(demisto.params().get('dontFormatSublists', False)):` over and over just create a variable and use it (also make it positive):
```python
should_format_sublists = not argToBoolean(demisto.params().get('dontFormatSublists', False))
````